web/modules/custom/createpost/src/Plugin/Block/SubscribeFormButton.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Drupal\createpost\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * Provides a button subscribe to newsletter.
+ *
+ * @Block(
+ *   id = "createpost_subsribe_form_button",
+ *   admin_label = @Translation("Subscribe to Newsletter"),
+ * )
+ */
+class SubscribeFormButton extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+
+    return [
+      '#attached' => [
+        'library' => [
+          'core/drupal.dialog.ajax',
+          'core/jquery.form',
+          'createpost/global-styling',
+        ],
+      ],
+    ];
+  }
```

Comment:
Why this block is needed?

Reply:
I have used this block to attach libraries to my multi-step modal modal form

Reply:
You can do this directly in the form render array itself right?

Reply:
Yes I will add this to render array